### PR TITLE
[production] Release endpoint package 0.12.0 to production

### PR DESCRIPTION
**Pull request by jonathan-buttner**
Cherry-picking the commit from staging for endpoint v0.12.0.

### CODE REVIEW

**Comment by ruflin** [comment]
@jonathan-buttner I just merged https://github.com/elastic/package-storage/pull/186. Can you try to rebase this PR to see if it works now?

File: packages/endpoint/0.12.0/dataset/file/fields/fields.yml [hunk 1]
```diff
@@ -0,0 +1,705 @@
...
+
+      Example: For Beats this would be beat.id.'
+    example: 8a4f500d
+  - name: type
+    level: core
+    type: keyword
+    ignore_above: 1024
+    description: 'Type of the agent.
+
+      The agent type stays always the same and should be given by the agent used.
```
Comment (michaelolo24) [inline packages/endpoint/0.12.0/dataset/file/fields/fields.yml:56]: this has probably always been there and not actually related to your change, but the wordings a bit weird. `always stays the same` would be smoother

**Comment by jonathan-buttner** [comment]
Need to make additional changes so I'm going to skip this release.

**Pull request closed without merging**